### DISCUSSION TITLE
Can't install to emcc

### DISCUSSION THREAD

**Discussion by TweedLee** [devmfc/debian-on-amlogic#16] [Q&A] [answered]
Many thanks to Devmfc for making these images, they're probably the only Linux images available for the x96 X4.
Forgive me for being a Linux idiot, I failed to run the "install to emmc" command.
It's appreciated if anyone could help me with the next step!
*Now the box said "unable to locate fdisk"
![img_v3_0263_55a379e6-10f1-49ee-bbda-c673612c220g](https://github.com/devmfc/debian-on-amlogic/assets/94418133/e214681e-8ab7-4db3-865a-66af7fe23d76)

**Comment by TweedLee** [comment]
Update: I have written the system to EMMC, but I am unable to boot without the SD card.
![image](https://github.com/devmfc/debian-on-amlogic/assets/94418133/986822b0-e1f9-4fd5-8b25-6dc6471e95cb)

**Reply by devmfc** [reply]
Great!
You have to run  `./aml-multiboot-setup.sh` for that, have you done that?

**Reply by TweedLee** [reply]
Thank you devmfc! Yes I have run ./aml-multiboot-setup.sh
![image](https://github.com/devmfc/debian-on-amlogic/assets/94418133/e8196daf-c369-4c30-b2c4-ab7516988447)
(I changed the boot.config in the sd card so that boot dev=mmcblk1p1). 
The sd card can be removed after the system boots up, but the system can't be booted up without the sd card. It will get stuck on the startup screen. 
![image](https://github.com/devmfc/debian-on-amlogic/assets/94418133/bc57e28d-7d60-471b-83e6-3caf4b836a9a)

**Reply by devmfc** [reply, marked as answer]
You might have a newer u-boot. The newer u-boots seems to ignore MBR/DOS partition tables. So it can't find your /boot partition on EMMC. When Linux is booted it does see the partition table and mount your root partition from EMMC. I don't have a solution for that yet, but there is a workaround. For that to confirm I need to have either a serial log or the android image you used to flash your device.

**Reply by TweedLee** [reply]
Thank you for your reply! That gives me an idea of the problem I'm having. But unfortunately, I don't have a backup of the original Android image. I'll go ahead and boot it via TF card. Thanks again!

**Reply by LITUATUI** [reply]
You can try [slimBOXtv](https://slimboxtv.ru/x96-x4/) in the internal memory.
